apps/studio/pages/api/ai/edge-function/complete.ts

```diff
@@ -64,20 +65,28 @@ async function handlePost(req: NextApiRequest, res: NextApiResponse) {
       : { result: [] }
 
     const result = await streamText({
-      model: openai('gpt-4o-mini-2024-07-18'),
+      model: bedrock('us.anthropic.claude-3-7-sonnet-20250219-v1:0'),
       maxSteps: 5,
       tools: getTools({ projectRef, connectionString, authorization, includeSchemaMetadata }),
```

Comment:
are we making an unnecessary call up above? https://github.com/supabase/supabase/pull/35318/files#diff-1b9db5c94dc0b211d2ebee03daebbf8217d108ab3027a851971ce5bf56cddaacR52-R65